File: app/src/main/java/jp/ac/dendai/c/jtp/adventuresample/scene/Ending.java

package jp.ac.dendai.c.jtp.adventuresample.scene;
import jp.ac.dendai.c.jtp.adventuresample.R;
public class Ending extends AbstractScene {
    @Override
    public int getImageId() {
        return R.drawable.ending;
    }
    @Override
    public int getMessageId() {
        return R.array.messageending;
    }
    @Override
    public int getQuestionId() {
        return 0;
    }
    @Override
    public GameState next(int no) {
        return null;
    }
    @Override
    public int getDateId() {
        return R.string.dateending;
    }
}

File: app/src/main/java/jp/ac/dendai/c/jtp/adventuresample/scene/Ending.kt
package jp.ac.dendai.c.jtp.adventuresample.scene

import jp.ac.dendai.c.jtp.adventuresample.R

class Ending : AbstractScene() {
    override val imageId = R.drawable.ending

    override val messageId = R.array.messageending

    override val questionId = 0

    override fun next(no: Int): GameState? = null

    override val dateId = R.string.dateending

    override val musicId = R.raw.owarininokorumono
}